Python/tests/package_management_file_test.py

```diff
@@ -98,21 +107,13 @@ def test_install_basic_zip_package():
 
     _create(module_name=module_name, package_file=package, class_to_check="ClassA")
 
-
-def test_install_basic_zip_package_different_name():
-    package = os.path.join(path_to_packages, "testpackageA-0.0.1.zip")
-    module_name = "testpackageA"
-
-    _remove_all_new_packages(pkgmanager)
-    
-    _create(module_name=module_name, package_file=package, class_to_check="ClassA")
-
-
+# Test some wheel files
+#
 def test_install_whl_files():
-    packages = ["webencodings-0.5.1-py2.py3-none-any.whl", "html5lib-1.0.1-py2.py3-none-any.whl",

```

Comment:
any reason why we are removing these packages?

Reply:
I trimmed the size of each test to make it faster by removing larger packages or packages with dependencies if possible.

Also, webencodings is a system package, so testing installing it doesn't really work.